src/experimentation/handler.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { badRequest, noContent } from '@adobe/spacecat-shared-http-utils';
+import { hasText, isObject } from '@adobe/spacecat-shared-utils';
+import { convertToCSV } from '../support/utils.js';
+
+import {
+  uploadSlackFile, postSlackMessage, markdown, section,
+} from '../support/slack.js';
+
+export function buildExperimentationSlackMessage(url, auditResult) {
+  const blocks = [];
+  blocks.push(section({
+    text: markdown(`For *${url}*, ${auditResult.length} experiments have been run in the *last week*.\n More information is below :`),
+  }));
+  for (let i = 0; i < Math.min(3, auditResult.length); i += 1) {
+    const topLine = section({
+      text: markdown(`:arrow-red2: * Experiment - ${auditResult[i].experiment}| Period - ${auditResult[i].time5} to ${auditResult[i].time95} | Confidence - ${auditResult[i].p_value} | Events - ${auditResult[i].variant_experimentations} | Conversion - ${auditResult[i].variant_conversions} | Conversion Rate - ${auditResult[i].variant_conversion_rate} *`),
+    });
+    blocks.push(topLine);
+  }
+  return blocks;
+}
+
+function isValidMessage(message) {
```

Comment:
I would move this function to utils.js as well because it's used by more than 2 handlers